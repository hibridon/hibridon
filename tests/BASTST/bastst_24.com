inp=bastst_24.inp
out=bastst_24.out
run
save=save_24.inp