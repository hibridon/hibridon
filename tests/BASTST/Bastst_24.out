      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   = -30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** SPH-TOP system parameters:
 NTERM  =   4        IOP    =   1        JMAX   =  10
 BROT   =  5.241     D-J    = 1.1086E-04 D-K    = 4.4250E-06
 LAMMIN:    3   4   6   7
 LAMMAX:    3   4   6   7
 MPROJ:     0   0   0   0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: 10; JOUT:  0   1   2   3   4   5   6   7   8   9
 INDOUT:    1   2
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 07:11
CMAKE BUILD TYPE: Release
CODE GIT REVISION: 865fa9b03406cbd6da8c4193de3a9362b8dacc65
SYS: Darwin 22.2.0 x86_64
COMPILER: /usr/local/bin/gfortran
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 07:19:34 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS IN POTENTIAL =  4

 **  CC SPHERICAL TOP **
     RMU=  16.4700  BROT=  5.241
     DJ= 0.110864E-03     DK= 0.442500E-05
     IOP= 1  E=5000.00  JTOT=  20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=  -30.00

  CHANNEL LIST SORTED BY ENERGY
   N   J  IS  EPS  L  EINT(CM-1)
   1   0   1   1  20     0.000
   2   3   1   1  17    62.892
   3   3   1   1  19    62.892
   4   3   1   1  21    62.892
   5   3   1   1  23    62.892
   6   4   1   1  16   104.821
   7   4   1   1  18   104.821
   8   4   1   1  20   104.821
   9   4   1   1  22   104.821
  10   4   1   1  24   104.821
  11   6   1  -1  15   220.123
  12   6   1  -1  17   220.123
  13   6   1  -1  19   220.123
  14   6   1  -1  21   220.123
  15   6   1  -1  23   220.123
  16   6   1  -1  25   220.123
  17   6   2   1  14   220.123
  18   6   2   1  16   220.123
  19   6   2   1  18   220.123
  20   6   2   1  20   220.123
  21   6   2   1  22   220.123
  22   6   2   1  24   220.123
  23   6   2   1  26   220.123
  24   7   1   1  13   293.498
  25   7   1   1  15   293.498
  26   7   1   1  17   293.498
  27   7   1   1  19   293.498
  28   7   1   1  21   293.498
  29   7   1   1  23   293.498
  30   7   1   1  25   293.498
  31   7   1   1  27   293.498
  32   8   1   1  12   377.355
  33   8   1   1  14   377.355
  34   8   1   1  16   377.355
  35   8   1   1  18   377.355
  36   8   1   1  20   377.355
  37   8   1   1  22   377.355
  38   8   1   1  24   377.355
  39   8   1   1  26   377.355
  40   8   1   1  28   377.355
  41   9   1   1  11   471.693
  42   9   1   1  13   471.693
  43   9   1   1  15   471.693
  44   9   1   1  17   471.693
  45   9   1   1  19   471.693
  46   9   1   1  21   471.693
  47   9   1   1  23   471.693
  48   9   1   1  25   471.693
  49   9   1   1  27   471.693
  50   9   1   1  29   471.693
  51   9   2  -1  12   471.693
  52   9   2  -1  14   471.693
  53   9   2  -1  16   471.693
  54   9   2  -1  18   471.693
  55   9   2  -1  20   471.693
  56   9   2  -1  22   471.693
  57   9   2  -1  24   471.693
  58   9   2  -1  26   471.693
  59   9   2  -1  28   471.693
  60  10   1   1  10   576.514
  61  10   1   1  12   576.514
  62  10   1   1  14   576.514
  63  10   1   1  16   576.514
  64  10   1   1  18   576.514
  65  10   1   1  20   576.514
  66  10   1   1  22   576.514
  67  10   1   1  24   576.514
  68  10   1   1  26   576.514
  69  10   1   1  28   576.514
  70  10   1   1  30   576.514
  71  10   2  -1  11   576.514
  72  10   2  -1  13   576.514
  73  10   2  -1  15   576.514
  74  10   2  -1  17   576.514
  75  10   2  -1  19   576.514
  76  10   2  -1  21   576.514
  77  10   2  -1  23   576.514
  78  10   2  -1  25   576.514
  79  10   2  -1  27   576.514
  80  10   2  -1  29   576.514
ILAM=  1 LAMNUM(ILAM) =    500
ILAM=  2 LAMNUM(ILAM) =    732
ILAM=  3 LAMNUM(ILAM) =    998
ILAM=  4 LAMNUM(ILAM) =   1129
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS  3359
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO   11.1 %
      CURRENT DATE:  Mar 19 07:19:34 2023
 ===============================================================================
